picard/ui/mainwindow.py

```diff
@@ -1087,3 +1107,11 @@ def paste(self):
             target = selected_objects[0]
         self.tagger.paste_files(target)
         self.paste_action.setEnabled(False)
+
+    def check_for_update(self):
+        self.tagger.updatecheckmanager.check_update(show_always=True, update_level=config.setting["update_level"], callback=update_last_check_date)
+
+def update_last_check_date(is_success):
+    log.debug('Processing update_last_check_date callback with argument: %s' % (is_success,))
```

Comment:
I'd remove this `log.debug()` in final version of the PR, not very useful imho.
Instead be sure errors are logged in case of non-success ;)

Reply:
Will do, but I have a question regarding log entries.  Should they all be translated?  The reason I ask is because I see them translated in some places and not in others in the existing code.  I'd like to do it properly in any new code or changes that I make.

Reply:
Well, the thing is that error/warning messages should be translated because they are targetted to the user mainly.
For debug messages, it's not that great, because we'll end with debug logs (provided by users having problems) with messages in other languages. We perhaps need to rethink the whole thing, especially now we have a nice log dialog, with an option to save an english-only log for sharing and still have translated messages for users.
Another concern is the fact it will be hard to translate debug messages for translators without a dev background, and those messages change often (as they are used mainly by devs).

So, for now, don't mark or translate debug levell messages, but do for anything not debug.
Which is more or less what we do since years.

I'd prefer to have more languages supported, so keeping the number of strings to translate to acceptable level is much better regarding this goal.